### PR TITLE
#36 Added formula 6.1 and included .idea/ folder (Pycharm) to .gitignore

### PR DESCRIPTION
**Pull request by johan-tuls**
## Description

- [ ] Added formula 6.1
image
- [ ] Added .idea/ to .gitignore

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_6_ultimate_limit_state/formula_6_1.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        """[V_Rd] Design shear resistance of an element with shear reinforcement.
+
+        NEN-EN 1992-1-1+C2:2011 art.6.1(2) - Formula (6.1)
+
+        Parameters
+        ----------
+        v_rd_s : KN
+            [V_Rd,s] Design shear resistance of an element with shear reinforcement [kN].
+        v_ccd : KN
+            [V_ccd] Design value of the shear force component in the compression area in case of a change in height [kN].
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_6_ultimate_limit_state/formula_6_1.py:29]: Vccd

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_6_ultimate_limit_state/formula_6_1.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        self.v_td = v_td
+
+    @staticmethod
+    def _evaluate(
+        v_rd_s: KN,
+        v_ccd: KN,
+        v_td: KN,
+    ) -> KN:
+        """Evaluates the formula, for more information see the __init__ method"""
+        if v_rd_s < 0:
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_6_ultimate_limit_state/formula_6_1.py:45]: I suppose shear could be either negative or positive.

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_6_ultimate_limit_state/formula_6_1.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+        Parameters
+        ----------
+        v_rd_s : KN
+            [V_Rd,s] Design shear resistance of an element with shear reinforcement [kN].
+        v_ccd : KN
+            [V_ccd] Design value of the shear force component in the compression area in case of a change in height [kN].
+        v_td : KN
+            [V_td] Design value of the shear force component of the tensile force in the reinforcement in case of a change in height [kN].
+        """
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_6_ultimate_limit_state/formula_6_1.py:32]: add Return ;-)

**Review by egarciamendez — CHANGES_REQUESTED** [review]

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_6_ultimate_limit_state/formula_6_1.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+    def __init__(
+        self,
+        v_rd_s: KN,
+        v_ccd: KN,
+        v_td: KN,
+    ) -> None:
+        """[V_Rd] Design shear resistance of an element with shear reinforcement.
+
+        NEN-EN 1992-1-1+C2:2011 art.6.1(2) - Formula (6.1)
```
[the remark this answers is not in the document]
Comment (johan-tuls) [reply blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_6_ultimate_limit_state/formula_6_1.py:22]: added docstring

**Review by egarciamendez — APPROVED** [review]
🚀 let's merge

**Pull request merged**